src/test/java/UiStub.java

```diff
@@ -0,0 +1,22 @@
+import lebron.Lebron;
```

Comment:
Shouldn't every class be part of some package? Refer to the Java coding standard (Package and Import Statements section) regarding this.